ocs_ci/ocs/utils.py

```diff
@@ -1257,6 +1257,29 @@ def _collect_ocs_logs(
                 acm_mustgather_path, acm_mustgather_image, cluster_config=cluster_config
             )
 
+        # Check if subctl exists
+        # If we have deployed submariner then we should have binary in the
+        # path else submariner may not be in the picture and we will skip log collection
+        try:
+            run_cmd("subctl -h")
+        except FileNotFoundError:
+            log.warning(
+                "Subctl binary doesn't exists, checking if submariner pods exists..."
+            )
+            out = exec_cmd("oc get pods -n submariner-operator")
+            if out.stderr and ("No resources" in out.stderr):
+                log.warning(
+                    "Not able to find submariner resources, skipping submariner log collection"
+                )
+                return
+            else:
+                # submariner resources exists but subctl binary missing
+                # download subctl
+                # Importing here to avoid circular imports issue and tox failure
+                from ocs_ci.deployment.acm import download_subctl_binary
+
+                download_subctl_binary()
```

Comment:
Get clarity about the US/DS submariner and we need to pass here probably the value from config and also probably need to have the way to get the DS subctl as well - if it's not possible at the moment, we need to as W/A use US subctl only but at some point we should implement download subctl also for DS.